### PR TITLE
[Model] Add memory swap path property @open sesame 10/17 16:08

### PR DESCRIPTION
**Pull request by jihochu**
MemorySwapPath property is added to define path for swap files.
If it's not defined, it creates files in crurent directory.

Signed-off-by: Jiho Chu <jiho.chu@samsung.com>

### CODE REVIEW

File: nntrainer/models/neuralnet.cpp [hunk 1]
```diff
@@ -22,6 +22,7 @@
  */
 
 #include "layer_context.h"
+#include "model_common_properties.h"
```
Comment (lhs8928) [inline nntrainer/models/neuralnet.cpp:25]: neuralnet.h includes <model_common_property.h>

File: nntrainer/tensor/cache_pool.h [hunk 2]
```diff
@@ -14,9 +14,9 @@
 #ifndef __CACHE_POOL_H__
 #define __CACHE_POOL_H__
 
-#include <mutex>
-#include <memory_pool.h>
 #include <list>
+#include <memory_pool.h>
+#include <mutex>
```
Comment (lhs8928) [inline nntrainer/tensor/cache_pool.h:19]: Could you seperate nntrainer's header file and stl header file?

Comment (jihochu) [reply nntrainer/tensor/cache_pool.h:19]: thanks for checking

**Review by jijoongmoon — APPROVED** [review]
LGTM

**Review by lhs8928 — APPROVED** [review]
LGTM

**Pull request merged**